src/backlight/asset/currency.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class Currency(Enum):
+    USD = 1
+    JPY = 2
+    EUR = 3
+    GBP = 4
```

Comment:
Please add ccy at least here. https://www.euronextfx.com/listings

 And order should be defined by https://en.wikipedia.org/wiki/Currency_pair#Base_currency .